### PR TITLE
Feb Foundation community call

### LINKED ISSUE #140: February 2021 Foundation for Public Code Community call
This call will take place on 18 Feb 2021. The proposed topic is a public demo of the new publicode.net website and wireframes, led by @angelplasma and @ElenaFdR.

### PR DESCRIPTION
**Pull request by ElenaFdR**
Closes #140.

### CODE REVIEW

**Review by Ainali — CHANGES_REQUESTED** [review]
The name of the blog post file seems to have been partly lost.

File: _posts/2021-02-25- [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+
+
+# Notes: February 2021 Foundation for Public Code community call 
+
+In this community call, we talked through our thinking about [publiccode.net](https://publiccode.net/) as product ([pull request #169](https://github.com/publiccodenet/publiccode.net/pull/169)), and demo'd [our wireframes](https://www.figma.com/file/Mcz4evZVI4rRwv3quXtKGC/Site-Map-Wireframe-exploration-Foundation-for-Public-Code?node-id=177%3A0).
+
+
+## Attendees
+
+Alba Roza
```
Comment (Ainali) [inline _posts/2021-02-25-:18]: Was this intended to be an unordered list?

Comment (ElenaFdR) [reply _posts/2021-02-25-]: Fixed!

**Comment by ElenaFdR** [comment]
Travis is breaking on the Jekyll theme.

File: _posts/2021-02-25- [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+Jan Ainali
+Laura Scheske
+Martin Dias d'Ullois
+Rasmus Frey
+Ryan Conlan (guest speaker)
+
+   
+## Short update from the Foundation for Public Code
+
+- [Signalen is in incubation](https://blog.publiccode.net/news/2021/02/04/signalen-is-in-incubation.html) to fully meet the Standard for Public Code!
```
Comment (Ainali) [inline _posts/2021-02-25-]: I am not sure here, but does the first exclamation mark imply we should have periods on the other bullet points? Right now it looks a bit odd to me.

**Review by Ainali — APPROVED** [review]
Good to go!

**Pull request merged**